cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/DataPrepBrowserStore/index.js

```diff
@@ -280,6 +295,12 @@ const bigquery = (state = defaultBigQueryValue, action = defaultAction) => {
         datasetId: action.payload.datasetId,
         tableList: action.payload.tableList
       };
+    case Actions.SET_ERROR:
+      return {
+        ...state,
+        info: objectQuery(action, 'payload', 'info') || state.info,
+        loading: false
+      };
```

Comment:
Since there is already a specific reducer for error, are these error actions needed for the other reducers? Seems like it's doing the same thing over and over. Might be better to just clump everything related to errors into one reducer

Reply:
So I still need to handle this action for all the reducers, because each connection has its own `loading` state. However I have removed the `info` property handling since we aren't actually including it in the payload for this action.